interpreter/value_ephemeral_reference.go

```diff
@@ -138,6 +138,15 @@ func (v *EphemeralReferenceValue) GetMember(context MemberAccessibleContext, loc
 	return getMember(context, v.Value, locationRange, name)
 }
 
+func (v *EphemeralReferenceValue) GetMethod(
+	context MemberAccessibleContext,
+	locationRange LocationRange,
+	name string,
+) FunctionValue {
+	// TODO:
+	return nil
```

Comment:
Does this have to dereference like in `GetMember` and call `GetMethod` on the dereferenced value?

Reply:
Turned out that the references behave a bit differently. `GetMember` already returns the members of the dereferenced-value. But for references, the builtin-functions (such `getType`, `isInstance`) must also be returned on the dereferenced-value, but the `GetMember` implementations of other values doesn't really return those (for other values they are returned at the interpreter, when visiting member-access expression.)

So had to explicitly return the builtin functions here, for references. Added in:
https://github.com/onflow/cadence/pull/3886/commits/a08e05adbb877a96f4ec4ae8726624b5e097bec1